vaultwarden/umbrel-app.yml

```diff
@@ -2,7 +2,7 @@ manifestVersion: 1
 id: vaultwarden
 category: files
 name: Vaultwarden
-version: "1.25.0"
+version: "1.29.1"
```

Comment:
Looks like this will be the first update we've done of Vaultwarden, so we can now add a `releaseNotes` section to the `umbrel-app.yml`.

It can be a bit tricky trying to decide what to include in the release notes; the amount of detail really depends on the situation. Sometimes the release notes are just direct copies of the the official release notes, like this Torq example submitted by Torq themselves:
https://github.com/lncapital/torq/releases/tag/v1.0.1
<img width="1139" alt="image" src="https://github.com/getumbrel/umbrel-apps/assets/85373263/ede1aaf5-ed2f-493c-98af-bb727ae17ebc">

and other times we simplify the release notes if we are updating several versions at once, like this Plex example:
<img width="923" alt="image" src="https://github.com/getumbrel/umbrel-apps/assets/85373263/192ee0bc-4cc1-4dca-8943-4564e0c8758e">

For this update where we are moving forward several updates (1.25.0 all the way to 1.29.1), so it might make sense to just do a simplification like this:

```yaml
releaseNotes: >-
  This release updates Vaultwarden from v1.25.0 to v1.29.1. It includes many bug fixes and performance improvements, as well as the following new features:
  
  - Support for Forward Email
  
  - WebSocket notifications now work via the default HTTP port
  
  - Support for Argon2 key derivation

  - and more!


  The full release notes are available at https://github.com/dani-garcia/vaultwarden/releases
```

What do you think?

Reply:
Appreciate your suggestions! I think your proposal to simplify the release notes makes sense. Considering the multiple versions being covered, this will help keep the release notes concise while highlighting the key changes